client/my-sites/checkout/checkout-thank-you/domains/domain-thank-you.tsx

```diff
@@ -52,26 +56,36 @@ const DomainThankYou: React.FC< DomainThankYouContainerProps > = ( {
 		};
 	}, [ dispatch ] );
 
-	const renderHeader = () => {
+	const renderHeader = ( isLaunchpadEnabled: boolean, siteIntent: string ) => {
+		const buttonProps = isLaunchpadEnabled
+			? {
+					onClick: () =>
```

Comment:
`NB`: @Sharpirate wondering why we're using `onClick` instead of another `href`?

Reply:
I wasn't sure how to use `href` while making sure to respect the current environment (calypso or wordpress)